### PR TITLE
add the first weekly progress report

### LINKED ISSUE #239: Weekly Progress Report
### Issue Description

[The Weekly Progress Report](https://github.com/bounswe/bounswe2023group9/wiki/Weekly-Progress-Report-%231-%7C-3.10.2023) for the first lab meeting should be prepared.

### Deadline for the Issue

05.10.2023

### PR DESCRIPTION
**Pull request by uyduranomer**
this pull request includes the first weekly progress report. the merge of this pull request closes #239

### CODE REVIEW

**Review by hakanaktas0 — APPROVED** [review]
Everything we discussed in the meeting is included. Nice work.

File: reports/lab_report_01.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+| --- | --- | --- | --- | --- |
+| Reverify the project requirements | [#240](https://github.com/bounswe/bounswe2023group9/issues/240) | Team | 10.10.2023 | 1hr |
+| Reverify the UML diagrams if needed | -- | Bengisu, Zülal, Leyla | 10.10.2023 | 1hr |
+| Update wiki pages if needed | -- | Leyla | 10.10.2023 | 1hr |
+| Decide on what technologies are going to be used for the front-end | [#249](https://github.com/bounswe/bounswe2023group9/issues/249) | Front-end team | 10.10.2023 | 1.5hr |
+| Decide on what technologies are going to be used for the back-end | [#248](https://github.com/bounswe/bounswe2023group9/issues/248) | Back-end team | 10.10.2023 | 1.5hr |
+| Prepare the project plan | [#243](https://github.com/bounswe/bounswe2023group9/issues/243) | Team | 10.10.2023 | 3.5hr |
+| Prepare the Responsibility Assignment Matrix (RAM) | [#246](https://github.com/bounswe/bounswe2023group9/issues/246) | Hakan, Mehmet, Arda, Abdullah | 10.10.2023 | 2hr |
+| Adding Personal Wiki Pages | [#245](https://github.com/bounswe/bounswe2023group9/issues/245), [#237](https://github.com/bounswe/bounswe2023group9/issues/237) | Bengisu, Ömer Faruk | 10.10.2023 | 1hr
```
Comment (Brunettow) [inline reports/lab_report_01.md:19]: I realized that this line was missing. I added it.

**Review by Brunettow — APPROVED** [review]
I fixed the missing line. Everything looks fine now.

**Comment by uyduranomer** [comment]
This pull request has been reviewed and approved by 2 other member. I am mergeing this pull request.

**Pull request merged**